public/static/locales/en/editProfile.json

```diff
@@ -30,5 +30,9 @@
   "publicAccount": "Public Account",
   "accountPrivacyChangeText": "In order to create a profile widget you need to have a public account. By clicking continue you change your privacy settings to public.",
   "continue": "Continue",
-  "cancel": "Cancel"
+  "cancel": "Cancel",
+  "termAndConditionPart1":"I agree to the",
```

Comment:
From the perspective of a translator this is difficult and using one resource with a link placeholder would be better. But I am not sure if we use i18n already in a way to support this e.g. with a Trans component.

Reply:
Change this towards using the i18n Trans component to support links.